### PR TITLE
Added host_verify test scenario for windows to CI/CD.

### LINKED ISSUE #2145: Host_verify library for Windows.
Child of #2109.

1. Adding host_verify library for windows.
2. Unit tests for linux/windows.
3. CI/CD setup.

### PR DESCRIPTION
**Pull request by jazzybluesea**
1. Updated CI/CD to include host_verify test running in a windows nonSGX VM.
2. Moved cert generation used in host_verify test to Jenkinsfile from cmake.
3. Fixed memory leak in oecert.

Fixes #2145

### CODE REVIEW

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -271,7 +295,25 @@ def ACCHostVerificationTest(String version, String build_type) {
...
+                    cleanWs()
+                    checkout scm
+                    unstash "linux_host_verify-${build_type}-${BUILD_NUMBER}"
+                    powershell 'Copy-Item -Recurse C:\\openenclave\\prereqs\\nuget ${env:WORKSPACE}\\prereqs'
+                    dir('build') {
+                        bat """
+                            vcvars64.bat x64 && \
+                            cmake.exe ${WORKSPACE} -G Ninja -DBUILD_ENCLAVES=OFF -DUSE_LIBSGX=OFF -DCMAKE_BUILD_TYPE=${build_type} -Wdev && \
+                            ninja -v && \
+                            ctest.exe -V -C ${build_type} -R host_verify --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
```
Comment (johnkord) [inline .jenkins/Jenkinsfile]: This is really quite nice work figuring this out, Sergio. Nice job!

File: docs/GettingStartedDocs/GettingStarted.Windows.md [hunk 2]
```diff
@@ -215,6 +215,12 @@ It is available from [nuget.org](https://www.nuget.org/packages/Azure.DCAP.Windo
 nuget.exe install Azure.DCAP.Windows -ExcludeVersion -Version 0.0.2 -OutputDirectory C:\openenclave\prereqs\nuget
 ```
 
+NOTE: After instaling the Azure.DCAP.Windows package you will need to install the dcap_quoteprov.dll to the desired
```
Comment (johnkord) [inline docs/GettingStartedDocs/GettingStarted.Windows.md]: instaling -> installing

Comment (jazzybluesea) [reply docs/GettingStartedDocs/GettingStarted.Windows.md]: I talked Radhika and confirmed that this extra note is not necessary.

File: .jenkins/Jenkinsfile (hunk 1 above)
Comment (jazzybluesea) [reply .jenkins/Jenkinsfile]: Thanks!

**Pull request merged**